src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -48,6 +49,27 @@ public LdpcvHttpPost(final String username, final String password) {
         super(username, password);
     }
 
+    /**
+     * 4.3 may disallow post
+     *
+     * @param uri The repository root URI
+     */
+    @Test(groups = { "MAY" })
+    @Parameters({ "param1" })
+    public void ldpcvMayDisallowPost(final String uri) {
+        final TestInfo info = setupTest("4.3", "ldpcvMayDisallowPost",
+                "Although an LDPCv is both a TimeMap and an LDPC, implementations MAY disallow POST requests.",
+                "https://fedora.info/2018/06/25/spec/#ldpcv-post",
+                ps);
+
+        // create the versioned resource
+        final Response createResponse = createVersionedResource(uri, info);
+
+        final URI timeMapURI = getTimeMapUri(createResponse);
+        final String allow = doOptions(timeMapURI.toString()).getHeader("Allow");
```

Comment:
This test is failing when it should be succeeding.  Replace what you have at 69-70 with 

```
confirmPresenceOfHeaderValueInMultiValueHeader("Allow", "POST", doOptions(timeMapURI.toString()));
```

Reply:
Section 4.3.3 states:
"Although an LDPCv is both a TimeMap and an LDPC, implementations MAY disallow POST requests"
This test is checking that POST may be disallowed. There is already https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpOptions.java#L155 which tests that servers MAY support POST to LDPCvs. I'm not sure if there's a point to testing the negative, but it seems to be what this section is allowing.

We could drop this test if that would make more sense.

Reply:
Ah - good point. My bad.   I think it can stay.